languages/Cobol.groovy

```diff
@@ -20,6 +20,11 @@ buildUtils.assertBuildProperties(props.cobol_requiredBuildProperties)
 def langQualifier = "cobol"
 buildUtils.createLanguageDatasets(langQualifier)
 
+if (props.runzTests == "True") {
```

Comment:
It appears, that the property is now tested twice, it is already part of `createLanguageDatasets` , where the language cobol_testDatasets is used.

https://github.com/IBM/dbb-zappbuild/blob/5b283e9a8a13558afa2f8924448e1727b9b525c0/utilities/BuildUtilities.groovy#L357
So, there is definitely a defect in the `createLanguageDatasets` method.

However, with the proposed changes, this section would no longer be required. So, I think we should remove it from the `createLanguageDatasets` method

Reply:
I have removed the section in BuildUtilities.groovy